tripper/backends/sparqlwrapper.py

```diff
@@ -80,18 +81,50 @@ def query(
         Returns:
             The return type depends on type of query:
               - SELECT: list of tuples of IRIs for each matching row
+              - CONSTRUCT: generator over triples
               - ASK: TODO
-              - CONSTRUCT, DESCRIBE: TODO
+              - DESCRIBE: TODO
         """
-        self.sparql.setReturnFormat(JSON)
-        self.sparql.setMethod(POST)
-        self.sparql.setQuery(query_object)
-        ret = self.sparql.queryAndConvert()
-        bindings = ret["results"]["bindings"]
-        return [
-            tuple(str(convert_json_entrydict(v)) for v in row.values())
-            for row in bindings
-        ]
+        query_type = self._get_sparql_query_type(query_object)
+        if query_type == "SELECT":
+            self.sparql.setReturnFormat(JSON)
+            self.sparql.setMethod(POST)
+            self.sparql.setQuery(query_object)
+            ret = self.sparql.queryAndConvert()
+            bindings = ret["results"]["bindings"]
+            return [
+                tuple(str(convert_json_entrydict(v)) for v in row.values())
+                for row in bindings
+            ]
+        if query_type == "CONSTRUCT":
+            self.sparql.setReturnFormat(RDFXML)
+            self.sparql.setMethod(POST)
+            self.sparql.setQuery(query_object)
+            graph = self.sparql.queryAndConvert()
+            return ((str(s), str(p), str(o)) for s, p, o in graph)
```

Comment:
We should test if the object is a literal and convert it to a `tripper.Literal` in that case.